examples/decoding/plot_oasis_vbm_space_net.py

```diff
@@ -0,0 +1,70 @@
+"""
+Voxel-Based Morphometry on Oasis dataset with Space-Net prior
+=============================================================
+
+Predicting age from gray-matter concentration maps from OASIS dataset
+"""
+# Authors: DOHMATOB Elvis
+#          FRITSCH Virgile
+
+
+### Load Oasis dataset ########################################################
+import numpy as np
+from nilearn import datasets
+dataset_files = datasets.fetch_oasis_vbm()
+age = dataset_files.ext_vars['age'].astype(float)
+age = np.array(age)
+gm_imgs = np.array(dataset_files.gray_matter_maps)
+
+### Split data into training set and test set
+from sklearn.utils import check_random_state
+from sklearn.cross_validation import train_test_split
+rng = check_random_state(42)
+gm_imgs_train, gm_imgs_test, age_train, age_test = train_test_split(
+    gm_imgs, age, train_size=.6, random_state=rng)
+
+### Sort test data for better visualization (trend, etc.)
+perm = np.argsort(age_test)[::-1]
+age_test = age_test[perm]
+gm_imgs_test = gm_imgs_test[perm]
+
+
+### Fit and predict ###########################################################
+from nilearn.decoding import SpaceNetRegressor
+import matplotlib.pyplot as plt
+from nilearn.plotting import plot_stat_map
+for penalty in ['tv-l1', 'graph-net']:
+    # To save time (because these are anat images with many voxels), we include
+    # only the 5-percent voxels most correlated with the age variable to fit.
+    # Also, we set memory_level=2 so that more of the intermediate computations
```

Comment:
Unless you want to re-run masking on the same data while changing a parameter like low pass filtering, `memory_level=2` is useless.